users/views.py

```
@@ -66,3 +68,33 @@ def perform_destroy(self, instance):
 
     def update(self, request, *args, **kwargs):
         raise MethodNotAllowed("You can only create or delete a follow")
+
+
+class SuggestedUsersView(GenericViewSet, ListModelMixin):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)